openebs/types/v1/types.go

```diff
@@ -16,15 +16,16 @@ limitations under the License.
 
 package v1
 
-//VsmSpec holds the config for creating a VSM
-type VsmSpec struct {
+//VolumeSpec holds the config for creating a VSM
+type VolumeSpec struct {
 	Kind       string `yaml:"kind"`
 	APIVersion string `yaml:"apiVersion"`
 	Metadata   struct {
 		Name   string `yaml:"name"`
 		Labels struct {
 			Storage      string `yaml:"volumeprovisioner.mapi.openebs.io/storage-size"`
```

Comment:
Can you raise an issue to send the Capacity as "openebs.io/capacity"
The storage-size should be deprecated in future & should remain for backward compatibility purposes.